CLI: orchestration commands (#306)
kyma-project/control-plane: docs/cli/commands/kcp_upgrade_kyma.md

      --dry-run                      Perform the orchestration without executing the actual upgrage operations for the Runtimes. The details can be obtained using the "kcp orchestrations" command.
      --parallel-workers int         Number of parallel workers to use in parallel orchestration strategy. By default the amount of workers will be auto-selected on control plane server side.
      --schedule string              Orchestration schedule to use. Possible values: "immediate", "maintenancewindow". By default the schedule will be auto-selected on control plane server side.

>       --strategy string              Orchestration strategy to use. The possible values are: parallel. (default "parallel")

Reviewer: ```suggestion
      --strategy string              Orchestration strategy to use. The default value is "parallel". 
```

Reviewer: ... or `The only possible value is "parallel".
